src/decays/H_SM_decays/decay_H_to_WW.cpp

```diff
@@ -0,0 +1,102 @@
+// special case for H -> W+ W-
+// TODO: implement higher order corrections
+
+template <>
+double CLASSNAME::get_partial_width<H, conj<W>::type, W>(
+   const context_base& context, typename field_indices<H>::type const& indexIn,
+   typename field_indices<conj<W>::type>::type const& indexOut1,
+   typename field_indices<W>::type const& indexOut2) const
+{
+
+   const double mHOS = context.physical_mass<H>(indexIn);
+   const double mWOS = context.physical_mass<W>(indexOut1);
+   const double x = Sqr(mWOS/mHOS);
+   double res = 0;
+
+   if ((x > 1.0 && flexibledecay_settings.get(FlexibleDecay_settings::offshell_VV_decays) != 0) ||
+       (4.*x > 1.0 && flexibledecay_settings.get(FlexibleDecay_settings::offshell_VV_decays) == 2)) {
+
+      // integrand
+      constexpr double GammaW = 2.085; //3*std::norm(ghWW)/(16.*Pi*mWOS);
+      struct my_f_params params = {mHOS, mWOS, GammaW};
+      gsl_monte_function G = { &hVV_4body, 2, &params };
+
+      // setup integration
+      gsl_rng_env_setup ();
+      double xl[2] = {0, 0};
+      double xu[2] = {Sqr(mHOS), Sqr(mHOS)};
+      // this gives relative error < 0.05%
+      constexpr size_t calls = 1'000'000;
+      double err;
+      const gsl_rng_type *T = gsl_rng_default;
+      gsl_rng *r = gsl_rng_alloc (T);
+      gsl_monte_miser_state *s = gsl_monte_miser_alloc (2);
+      gsl_monte_miser_integrate (&G, xl, xu, 2, calls, r, s,
+                                 &res, &err);
+
+      // clean-up
+      gsl_monte_miser_free (s);
+      gsl_rng_free (r);
+
+      // prefactor
+      const auto indices = concatenate(indexIn, indexOut2, indexOut1);
+      const auto ghWW =
+         Vertex<H, conj<W>::type, W>::evaluate(indices, context).value();
+
+      res *= 1./(16.*Cube(Pi))* std::norm(ghWW) * 0.25*Cube(mHOS)/Power4(mWOS);
+      err *= 1./(16.*Cube(Pi))* std::norm(ghWW) * 0.25*Cube(mHOS)/Power4(mWOS);
```

Comment:
These two lines look identical. Better to not re-calculate these expressions twice.

Reply:
Also done in 0639a17a3d2b914b83db6771d5401437c6673002